code/_onclick/click.dm

```diff
@@ -327,15 +327,18 @@
 		ML.pulled(src)
 
 /mob/living/CtrlClick(mob/user)
-	if(isliving(user) && Adjacent(user) && !user.incapacitated())
-		if(world.time < user.next_move)
-			return FALSE
-		var/mob/living/user_living = user
-		if(user_living.apply_martial_art(src, null, is_grab=TRUE) == MARTIAL_ATTACK_SUCCESS)
-			user_living.changeNext_move(CLICK_CD_MELEE)
-			return
-	else
-		..()
+	if(!isliving(user) || !Adjacent(user) || user.incapacitated())
+		return ..()
+
+	if(world.time < user.next_move)
+		return ..()
+
+	var/mob/living/user_living = user
+	if(user_living.apply_martial_art(src, null, is_grab=TRUE) == MARTIAL_ATTACK_SUCCESS)
+		user_living.changeNext_move(CLICK_CD_MELEE)
+		return TRUE
+
+	return ..()
 
 
 /mob/living/carbon/human/CtrlClick(mob/user)
```

Comment:
Mind tidying this one up as well?

Reply:
On it !

Reply:
Er, I meant if you'd mind putting early returns for /mob/living/carbon/human/CtrlClick() as well (it's p.much copypasta code anyway)

Reply:
Will do tomorrow then.